op-node/p2p/cli/load_signer.go

```diff
@@ -0,0 +1,33 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/urfave/cli"
+
+	"github.com/ethereum-optimism/optimism/op-node/flags"
+	"github.com/ethereum-optimism/optimism/op-node/p2p"
+)
+
+// TODO: implement remote signer setup (config to authenticated endpoint)
+// and remote signer itself (e.g. a open http client to make signing requests)
+
+// LoadSignerSetup loads a configuration for a Signer to be set up later
+func LoadSignerSetup(ctx *cli.Context) (p2p.SignerSetup, error) {
+	key := ctx.GlobalString(flags.SequencerP2PKeyFlag.Name)
+	if key != "" {
+		// Mnemonics are bad because they leak *all* keys when they leak.
```

Comment:
Mnemonics are easier to back up than hex strings

Reply:
Old code / todo, we should just have the signer integration and skip this debate

Reply:
Hex strings are bad because you must put them into your paste buffer when you want to use them

Reply:
agreed :)